.github/scripts/transform_dbt_test_results.py

```diff
@@ -108,16 +108,13 @@
 # Mapping that defines category names that should be reported for tests
 # based on their generics
 TEST_CATEGORIES = {
-    "macro.athena.test_accepted_range": "incorrect_values",
-    "macro.dbt_utils.test_accepted_range": "incorrect_values",
-    "macro.athena.test_accepted_values": "incorrect_values",
-    "macro.athena.test_not_accepted_values": "incorrect_values",
-    "macro.dbt_utils.test_not_accepted_values": "incorrect_values",
-    "macro.athena.test_unique_combination_of_columns": "duplicate_records",
-    "macro.dbt_utils.test_unique_combination_of_columns": "duplicate_records",
-    "macro.athena.test_not_null": "missing_values",
-    "macro.athena.test_is_null": "missing_values",
-    "macro.athena.test_res_class_matches_pardat": "class_mismatch_or_issue",
+    "test_accepted_range": "incorrect_values",
+    "test_accepted_values": "incorrect_values",
+    "test_not_accepted_values": "incorrect_values",
+    "test_unique_combination_of_columns": "duplicate_records",
+    "test_not_null": "missing_values",
+    "test_is_null": "missing_values",
+    "test_res_class_matches_pardat": "class_mismatch_or_issue",
```

Comment:
This list remains the same, I've just removed the `macro.<project_name>.*` prefix from each key and deduplicated the keys that previously had a `dbt_utils` version.